### PR TITLE
change the category link

### PR DESCRIPTION
**Pull request by yojana-tripathi**
correct the link of footer category ..

### CODE REVIEW

File: src/component/footer/Footer.jsx [hunk 1]
```diff
@@ -76,16 +76,16 @@ const Footer = () => {
...
-                  <Link to="/DI">
+                  <Link to="/DATA_INTERPRETATION">
                     <h6 className="liWrapper">Data Interpretation</h6>
                   </Link>
-                  <Link to="/LR">
+                  <Link to="/LOGICAL_REASONING">
                     <h6 className="liWrapper">Logical Reasoning</h6>
                   </Link>
-                  <Link to="/VARC">
+                  <Link to="/VERBAL_ABILITY_&_READING_COMPREHENSION">
```
Comment (thakursachin05) [inline src/component/footer/Footer.jsx:88]: link not working please check

**Review by thakursachin05 — COMMENTED** [review]
Minor comments

File: src/component/footer/Footer.jsx (hunk 1 above)
Comment (yojana-tripathi) [reply src/component/footer/Footer.jsx]: i change this link plz check

**Review by Ashutosh741 — APPROVED** [review]
LGTM

**Pull request merged**